core/jobs/batch_jobs/contributor_admin_stats_jobs.py

```diff
@@ -558,7 +560,22 @@ def transform_question_contribution_stats(
 
         entity_id = contributor_user_id
 
+        logging.info(
```

Comment:
@chris7716 Since this is in a Beam job, don't you want the logging statements to be processed within that as well, so that they show up in the Beam job logs? Otherwise there is not going to be a straightforward way for you to query this data.

You should probably show a successful local run that demonstrates that the relevant logs have been added. Probably also want to log the failing cases rather than all cases, right?

Reply:
I agree with your point and attached the proof for the new method.

Reply:
Proof seems good, but why do you still have the logging.info() ? Anything you want to log should be in the Beam job output otherwise it will not be discoverable/traceable and will clutter the logs.